pkg/controller/controller_binding.go

```diff
@@ -1477,6 +1509,7 @@ func (c *controller) pollServiceBinding(binding *v1beta1.ServiceBinding) error {
 			successInjectedBindResultReason,
 			successInjectedBindResultMessage,
 		)
+		binding.Status.UnbindStatus = v1beta1.ServiceBindingUnbindStatusRequired
```

Comment:
The Controller should not have to set the `UnbindStatus` to `Required` here. The `UnbindStatus` should have already been set before the bind request was sent.